include/z64.h

```diff
@@ -941,6 +943,11 @@ typedef struct {
     /* 0x03 */ u8 byte3;
 } ElfMessage; // size = 0x4
 
+typedef struct {
+    /* 0x00 */ u8 numTransitionActors;
+    /* 0x04 */ TransitionActorEntry* transitionActorList;
+} DoorContext;
```

Comment:
I think it would make more sense to name it TransitionActorContext or something like that since this is for transition actors in general, and we could probably shorten the name of its fields based on the context (eg. `numActors` and `list`)